code/parachain/node/src/chain_spec/dali.rs

```diff
@@ -77,5 +77,6 @@ pub fn genesis_config(
 		vesting: Default::default(),
 		lending: Default::default(),
 		liquidations: Default::default(),
+		ibc: dali_runtime::IbcConfig { asset_ids: vec![(1u128.into(), b"UNIT".to_vec())] },
```

Comment:
What's the meaning of 1u128.into() and b"UNIT".to_vec()? Can we perhaps refactor the asset_ids type to take in a struct with named fields, instead a tuple?